projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -969,7 +969,10 @@ function wpcom_launchpad_is_mobile_app_installed_visible() {
 		return false;
 	}
 
-	return true;
+	// TODO: We are hidding the task for now because we the completion logic
+	// is not fully implemented yet. We should make it return true once the
+	// completion logic is shipped.
+	return false;
```

Comment:
Might it be better to just `return false;` as the first line in the function so we're not even checking whether this is an Atomic site?

Reply:
I removed the logic entirely for now, the linter would not accept to have a return followed by code.